<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>462</width>
    <height>542</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>90</y>
     <width>401</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>APLIKASI JASA PEMINJAMAN ALAT MUSIK</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>130</y>
     <width>261</width>
     <height>231</height>
    </rect>
   </property>
   <property name="accessibleName">
    <string/>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Raised</enum>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="btnUser">
      <property name="text">
       <string>USER</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnPelanggan">
      <property name="text">
       <string>PELANGGAN</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnAlat">
      <property name="text">
       <string>ALAT</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnPeminjaman">
      <property name="text">
       <string>PEMINJAMAN</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnPembayaran">
      <property name="text">
       <string>PEMBAYARAN</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
